data/na/swagger/1-0-5.yaml

```diff
@@ -2370,6 +2550,18 @@ definitions:
           $ref: '#/definitions/Detail'
       validation:
         $ref: '#/definitions/CardValidation'
+  RecurringPaymentErrorResponse:
+    properties:
+      code:
+        type: integer
+        format: int32
```

Comment:
Should we mention possible values and what they means

Reply:
I followed the same stuff as payments, it just shows the fields and its types, there are no examples.

Reply:
yeah, I still think it's good information but I can't check the ERD right now to see if those data are in database or just hardcoded stuff. I'm ok if we decide not to add them.